### ISSUE TITLE
Use Case 14: Discussion

### ISSUE THREAD

**Issue by brianthomas** [astrodataformat/usecases#11]
Hi, thanks for this use case. I am wondering if structured data also might include things such as representation of errors which are essentially "objects"? Can this be added, or do you think this is a separate Use Case? I've started Requirement 19 based on this Use Case, and there are many possible structured data. The list I've started probably exceeds your Use Case. Again, can the use case be expanded to include these types, or should a separate use case be started?

**Comment by msdemlei** [comment]
Definitely errors are covered by these complex objects.

I've tried to make that a bit clearer in commit 69f736e, but I wonder if
I should add some more, possibly less physical, examples that came up in
various VO contexts.  VOResource, which is in active use, has lots of
cases (e.g.: pertaining references, which _should_ be a sequence of
(bibliographic string, type-of-reference)).  STC has more sweet
examples.  Provenance we don't really have yet, but already from W3C
provenance it's clear there's many use cases for sequences of complex
objects.

**Comment by brianthomas** [comment]
@msdemlei Yes, lets add more examples if its not too much trouble. I think that documenting a range of metadata is useful. I'd like to capture as many sub-requirements here as possible. Its one of the things which the FITS paper called out as a serious problem with that format (limited metadata/data capture)
